src/client.rs

```diff
@@ -129,20 +123,30 @@ impl Client {
     ///
     /// // get the index named "get_raw_index"
     /// let raw_index = client.get_raw_index("get_raw_index").await.unwrap();
-    /// assert_eq!(raw_index.uid, "get_raw_index");
+    /// assert_eq!(raw_index.get("uid").unwrap().as_str().unwrap(), "get_raw_index");
     /// # index.delete().await.unwrap().wait_for_completion(&client, None, None).await.unwrap();
     /// # });
     /// ```
     /// If you use it directly from an [Index], you can use the method [Index::fetch_info], which is the equivalent method from an index.
-    pub async fn get_raw_index(&self, uid: impl AsRef<str>) -> Result<JsonIndex, Error> {
-        Index::fetch_info(&self.index(uid.as_ref())).await
+    pub async fn get_raw_index(&self, uid: impl AsRef<str>) -> Result<Value, Error> {
+        let idx = self.index(uid.as_ref());
+        request::<(), Value>(
+            &format!("{}/indexes/{}", self.host, idx.uid),
```

Comment:
It is possible to use:

```suggestion
        request::<(), Value>(
            &format!("{}/indexes/{}", self.host, uid.as_ref()),
```

Reply:
Actually yes!